tests/e2e/scale/noobaa/test_hsbench.py

```diff
@@ -0,0 +1,57 @@
+import logging
+import pytest
+from ocs_ci.ocs import hsbench
+from ocs_ci.utility import utils
+from ocs_ci.framework.testlib import (
+    E2ETest, scale, ignore_leftovers
+)
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='function')
+def hsbenchs3(request):
+
+    hsbenchs3 = hsbench.HsBench()
+
+    def teardown():
+        hsbenchs3.cleanup()
+    request.addfinalizer(teardown)
+    return hsbenchs3
+
+
+@scale
+@ignore_leftovers
```

Comment:
Any specific reason we need leftovers marker

Reply:
No specific reason for this.  It can be removed.